tar/storage/storage_read.c

```diff
@@ -58,6 +62,42 @@ static sendpacket_callback callback_read_file_send;
 static handlepacket_callback callback_read_file_response;
 static int callback_read_file(void *, int, uint8_t *, size_t);
 
```

Comment:
Spurious blank line.

Reply:
That was somewhat intended, since the `cache_free` was going to be deleted in the following PR.  But yeah, on second thought I agree that I shouldn't be adding the blank line.